### PR TITLE
update-broken-test

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md
- [x] This change has been tested via the terms in TESTING.md

### CODE REVIEW

File: cypress/integration/disable-click.spec.ts [hunk 1]
```diff
@@ -3,8 +3,6 @@ describe("Disable click", () => {
     cy.visit("/cypress/disable-click");
     cy.get(".meridian-map-container").should("exist");
```
Comment (vercasson) [inline cypress/integration/disable-click.spec.ts:4]: I was going to remove this also, but I thought I should ask if you're doing this for a reason (like an implicit wait).

**Review by tarakreft — APPROVED** [review]

File: cypress/integration/disable-click.spec.ts (hunk 1 above)
Comment (wavebeem) [reply cypress/integration/disable-click.spec.ts:4]: Yeah I needed this for the test to pass consistently in CI

File: cypress/integration/disable-click.spec.ts [hunk 2]
```diff
@@ -3,8 +3,6 @@ describe("Disable click", () => {
...
     // Test all tags and placemarks
-    cy.get("[data-meridian-tag-id]").should("have.length", 3);
-    cy.get("[data-meridian-placemark-id]").should("have.length", 31);
```
Comment (wavebeem) [inline cypress/integration/disable-click.spec.ts:7]: Why did you remove these checks? I wanted to make sure all of the tags where present and disabled

Comment (wavebeem) [reply cypress/integration/disable-click.spec.ts:7]: I don't see how it's redundant. what if the "disabled" feature accidentally hid one of the placemarks? It doesn't seem super likely, but I thought about it.

it's probably not a big deal to remove it, but 🤷

Comment (vercasson) [reply cypress/integration/disable-click.spec.ts:7]: Cool. @wavebeem I do actually think both the filter tests (name and ID) could benefit from a count and the addition of resetting/removing of the filter and doing a follow-up count.

Comment (vercasson) [reply cypress/integration/disable-click.spec.ts:7]: ^ but wait on that until I comment out the second floor placemark (that appears on the first floor).
